Foundation/NSSwiftRuntime.swift

```diff
@@ -14,10 +14,16 @@ import CoreFoundation
 // This mimics the behavior of the swift sdk overlay on Darwin
 #if os(OSX) || os(iOS)
 @_exported import Darwin
-#elseif os(Linux)
+#elseif os(Linux) //// || os(Android) // Causes ambiguity in NSScanner.swift
```

Comment:
Glibc should be included here to make import Foundation be an implicit import of Glibc so I’ll change this. It causes ambiguity on FLT/DBL_MAX/MIN in NSScanner.swift

// these might be good to have in the stdlib
extension Float : _FloatLike {
    static var max: Float { return FLT_MAX }
    static var min: Float { return FLT_MIN }
}